scripts/test

```diff
@@ -22,7 +22,7 @@ echo "Did test-run-basics $?"
 echo "Did test-run-compat $?"
 
 . ./scripts/test-run-etcd
-echo "Did test-run-etcd $?"
+ echo "Did test-run-etcd $?"
```

Comment:
nit. I think there is an unintended space here

Reply:
Fixed